src/graphql/schema/collection/createCollectionQueryFieldEntries.ts

```diff
@@ -34,7 +34,7 @@ export default function createCollectionQueryFieldEntries (
             type: gqlConditionType,
           }],
         ],
-        getPaginatorInput: (headValue: mixed, args: { condition?: { [key: string]: mixed } }) =>
+        getPaginatorInput: (_headValue: mixed, args: { condition?: { [key: string]: mixed } }) =>
```

Comment:
It's convention for function parameters that aren't used; I hope the use of an underscored variable would be flagged during code review. It's a shame the `variable-name` lint rule can't be set to only `allow-leading-underscore` for unused function parameters.

If you prefer we can turn off the `noUnusedParameters` rule, but in my own code I've found that kind of rule has caught an issue from time to time where I've forgotten to implement a subtle part of the function.

https://github.com/Microsoft/TypeScript/wiki/What%27s-new-in-TypeScript#flag-unused-declarations-with---nounusedparameters-and---nounusedlocals